feature-libs/asm/components/asm-main-ui/asm-main-ui.component.ts

```diff
@@ -263,8 +267,25 @@ export class AsmMainUiComponent implements OnInit, OnDestroy {
     }
   }
 
-  andThen(_options: any) {
-    //navigate to other pages
+  protected handleDeepLinkParamsAfterStartSession(
+    parameters: AsmDeepLinkParameters
+  ) {
+    if (parameters.orderId) {
```

Comment:
maybe check `type` is more reasonable
this.asmComponentService.getSearchParameter('type')